opendevin/action/base.py

```diff
@@ -51,7 +51,7 @@ def executable(self) -> bool:
 @dataclass
 class NullAction(NotExecutableAction):
     """An action that does nothing.
-    This is used when the agent need to receive user follow-up messages from the frontend.
+    This is used when the agent need to receive user follow-up messages from the frontend or send an error.
```

Comment:
```suggestion
```

Honestly I think this gets used in 100 different ways--we should probably keep the description vague